dockers/docker-lldp/lldpmgrd

```diff
@@ -94,6 +94,11 @@ class LldpManager(daemon_base.DaemonBase):
         """
         port_desc = None
 
+        # Skip port name prefixed with "Inband". These are recycle ports exposed in PORT_TABLE for
+        # asic-to-asic communication in VOQ based chassis system. We do not configure LLDP on these.
+        if 'Inband' in port_name:
```

Comment:
Should it be `port_name.startswith('Inband')` to match only on prefix?

Reply:
As per Recycle port HLD PR discussion, it was recommended to use name starting with Ethernet. For inband interface, it will be Ethernet-Inband0. For recirc port it will be Ethernet-Recirc0. I'll change this to port_name.startswith("Ethernet-Inband") or use the inband_prefix() as recommended in next comment.

Reply:
Fixed.